config/sync/layout_library.layout.stanford_basic_page_full.yml

```diff
@@ -59,6 +59,36 @@ layout:
         additional: {  }
         weight: 0
     third_party_settings: {  }
+  -
+    layout_id: jumpstart_ui_one_column
+    layout_settings:
+      extra_classes: ''
+      centered: centered-container
+      columns: default
+      label: 'page description'
+    components:
+      5659aa07-22e4-461e-b826-00d505ba2d0c:
+        uuid: 5659aa07-22e4-461e-b826-00d505ba2d0c
+        region: main
+        configuration:
+          id: 'field_block:node:stanford_page:su_page_description'
```

Comment:
Ooh we missed that on the full width page.

Reply:
Yeah, it was tricky to remember that the layout isn't the same between default and full-width, but I think I got it right here.

Reply:
Did it have to be a whole other layout? could it just be added to the same section as the page title? it's just a little less confusing when editing.

Reply:
I made it match the way it was set up in the default layout, and they both use the same theming. Do you mean confusing when editing the layout, or confusing for content authors?

Reply:
Yeah the result is the same. i don't really care in the long run. 1 layout with 2 components vs 2 layouts with 1 component each. it's still the same outcome.

Reply:
So let's remove this and let's also remove the description from the regular node display.

Reply:
This is fixed.  Page description removed from both the default and full page layout.